### DISCUSSION TITLE
Lịch họp 8h 16/04/2023

### DISCUSSION THREAD

**Discussion by HoangNhu01** [HoangNhu01/eshop_mvc_monolithic#14] [Announcements]
Send leader all your product to complete the sprint 1

**Comment by TanK109** [comment]
okk

**Comment by HoangNhu01** [comment]
@TanK109 bạn chuẩn bị demo cho t phần FE những issue bạn close nhé, sử lại commet đi close lại thì chỉ rõ là commit bao nhiêu nhé

**Comment by HoangNhu01** [comment]
Complete 
- Require diagram
- Database
- Some feature screens
